ci/management/commands/generate_reports.py

```diff
@@ -126,11 +142,15 @@ def handle(self, *args, **kwargs):
 
         start_date = kwargs['start']
         end_date = kwargs['end']
-        file_name = kwargs['output_file']
+        output_file = kwargs['output_file']
+
+        email_recipients = kwargs['email_to']
+        email_sender = kwargs['email_from']
+        email_subject = kwargs['email_subject']
 
-        if not file_name:
```

Comment:
this was just a rename, the old variable was called `file_name` the new one is called `output_file` and if that one's not specified it will raise an exception.